I0626 15:03:13.173169 123783978506048 03_eval_finetuned.py:49] Loading finetuned model...
Traceback (most recent call last):
  File "/home/yunqiliu/octo/examples/03_eval_finetuned.py", line 115, in <module>
    app.run(main)
  File "/home/yunqiliu/anaconda3/envs/torch/lib/python3.11/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/yunqiliu/anaconda3/envs/torch/lib/python3.11/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/yunqiliu/octo/examples/03_eval_finetuned.py", line 50, in main
    model = OctoModel.load_pretrained(FLAGS.finetuned_path)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yunqiliu/octo/examples/octo/model/octo_model.py", line 266, in load_pretrained
    if checkpoint_path.startswith("hf://"):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'startswith'